Assets/Scripts/UI/BossHPBarManager.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class BossHPBarManager : MonoBehaviour
+{
+    [SerializeField]
+    private FinalBoss BossStats;
+    public Image Square;
+    public float healthAmount;
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        healthAmount = BossStats.HP;
+        Debug.Log(healthAmount);
+    }
+
+    // Update is called once per frame
+    public void Update()
+    {
+        //healthAmount = playerStats.CurrentHP;
+        if (Input.GetKeyDown("i"))
```

Comment:
should maybe wrap debug things like this in an editoronly define so it doesn't end up in final builds and mark with a todo to remove.